packages/components/organisms/f-checkout/src/components/Header.vue

```diff
@@ -108,6 +110,10 @@ export default {
     text-align: center;
 }
 
+.c-checkoutHeader-userTitle {
```

Comment:
not sure if this class name is good enough

Reply:
I think if this is the H1 asking the user to confirm their details then it's definitely good enough 🙂

Reply:
Any reason not to just call it `c-checkoutHeader-title`? Just protects it a little if the content of the h1 changes in the future (away from being a user title)

Reply:
Just seen that there's already a title called `c-checkoutHeader-title` so guessing that's why. Looking at the two titles, I'd probably make this class the `c-checkoutHeader-title` and change the classname of the other to be named closer to guest experience – `c-checkoutHeader-guestTitle`?

Reply:
yeah, that was the reason. The styles are a bit different so I needed a new class name. Your suggestion sounds good. Will change the class names.